src/app/vault/view.component.ts

```diff
@@ -48,4 +55,17 @@ export class ViewComponent extends BaseViewComponent implements OnChanges {
         this.platformUtilsService.eventTrack('View Password History');
         this.onViewCipherPasswordHistory.emit(this.cipher);
     }
+
+    copy(value: string, typeI18nKey: string, aType: string) {
+        super.copy(value, typeI18nKey, aType);
+        this.minimizeIfNeeded();
+    }
+
+    public async minimizeIfNeeded(): Promise<void> {
```

Comment:
don't need to specify `public`. Linter should be telling you this.

Reply:
Done.